pkgs/shelf_test_handler/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 ## 2.0.3-wip
 
-* Require Dart `^3.3.0`.
+* Require `shelf: ^1.1.0`
+* Require `test: ^1.25.15`
+* Require `sdk: ^3.5.0`.
```

Comment:
For similarity with other entries:

```suggestion
* Require Dart `3.5`.
```

Reply:
LOL! I was trying to explicitly make these more copy-paste so they are always consistent and minimal effort.

Uh...

I guess auto-merge decided for us.